### PR TITLE
Upstreaming romecrb: add msrs.rs

### PR DESCRIPTION
**Pull request by Rwantare**

### CODE REVIEW

File: src/mainboard/amd/romecrb/src/msr.rs [hunk 1]
```diff
@@ -0,0 +1,428 @@
+use core::fmt::Write;
+use x86_64::instructions::{rdmsr, wrmsr};
+/// Only read the MSR
```
Comment (rjoleary) [inline src/mainboard/amd/romecrb/src/msr.rs:3]: Please put a blank line before this line.

**Review by rjoleary — COMMENTED** [review]
Can you squash the commits?
Otherwise, everything else looks great!

File: src/mainboard/amd/romecrb/src/msr.rs (hunk 1 above)
Comment (rminnich) [reply src/mainboard/amd/romecrb/src/msr.rs:3]: should we follow the go style of doing this: 
/// read_msr reads an msr, compares it to a "known good" value, and prints a message, indicating
if matches or is different from that value.

File: src/mainboard/amd/romecrb/src/msr.rs [hunk 2]
```diff
@@ -0,0 +1,428 @@
...
+    let checked_value = rdmsr(address);
+    writeln!(w, " -- wrmsr: and got {:x}; \r", checked_value).expect("Failed to write!");
+}
+
+/// To be used on Write-Only MSRs
+fn _write_msr(w: &mut print::WriteTo, address: u32, value: u64) {
+    unsafe { wrmsr(address, value) }
+}
+
+pub fn msrs(w: &mut print::WriteTo) {
```
Comment (rminnich) [inline src/mainboard/amd/romecrb/src/msr.rs:28]: might as well document this one a little bit too.

**Review by rminnich — APPROVED** [review]

**Pull request merged**